metadata-patch.md

```diff
@@ -0,0 +1,195 @@
+
+Note:
+
+* When no values exist yet for a metadata key, you must add the key and value array with one
+  or more values.
+* When values already exist for a metadata key, it is necessary to specify the new value's
+  position at the end of the `path` (`/0` means first position, `/-` means last).
+* When creating new metadata values, if unspecified, the `language` and `authority` properties
+  default to `null`, and `confidence` defaults to `-1`.
+
+Example request, adding three values:
+
+```json
+[
+  {
+    "op": "add",
+    "path": "/metadata/dc.description",
+    "value": [ { "value": "Some description" } ]
+  },
+  {
+    "op": "add",
+    "path": "/metadata/dc.title/0",
+    "value": { "value": "Zeroth Title" }
+  },
+  {
+    "op": "add",
+    "path": "/metadata/dc.title/-",
+    "value": { "value": "Final Title", "language": "en_US" }
+  }
+]
+```
+
+New metadata state:
+
+```json
+{
+  "metadata": {
+    "dc.description": [
+      { "value": "Some description", "language": null, "authority": null, "confidence": -1 }
+    ],
+    "dc.title": [
+      { "value": "Zeroth Title", "language": null, "authority": null, "confidence": -1 },
+      { "value": "Initial Title", "language": null, "authority": null, "confidence": -1 },
+      { "value": "Final Title", "language": "en_US", "authority": null, "confidence": -1 }
+    ]
+  }
+}
+```
+
+## Removing metadata
+
+With the `remove` operation, you can delete:
+
+* All metadata with a given key (e.g. `path="/metadata/dc.description"`)
+* A single metadata value with a known position (e.g. `path="/metadata/dc.title/0"`)
+
+When only one value is present for a particular key, it is functionally equivalent to
+specify the path to the key or the value.
+
+Example request, removing two values:
+
+```json
+[
+  {
+    "op": "remove",
+    "path": "/metadata/dc.description"
+  },
+  {
+    "op": "remove",
+    "path": "/metadata/dc.title/0"
+  }
+]
+```
+
+New metadata state:
+
+```json
+{
+  "metadata": {
+    "dc.title": [
+      { "value": "Initial Title", "language": null, "authority": null, "confidence": -1 },
+      { "value": "Final Title", "language": "en_US", "authority": null, "confidence": -1 }
+    ]
+  }
+}
+```
+
+## Replacing metadata
+
+With `replace`, you can overwrite any of the following within a single operation
+of a PATCH request:
+
+* The entire set of all object metadata (`path="/metadata"`)
+* All metadata values for an existing key (e.g. `path="/metadata/dc.title"`)
+* A single existing metadata value (e.g. `path="/metadata/dc.title/0"`)
+* A single property of an existing value (e.g. `path="/metadata/dc.title/0/language"`)
+
+Example request, replacing the `value` and `language` of the first `dc.title`:
+
+```json
+[
+  {
+    "op": "replace",
+    "path": "/metadata/dc.title/0/value",
+    "value": "最後のタイトル"
+  },
+  {
+    "op": "replace",
+    "path": "/metadata/dc.title/0/language",
+    "value": "ja_JP"
+  }
+]
```

Comment:
I suggest to use a more compact/efficient example here:
```
[
  {
    "op": "replace",
    "path": "/metadata/dc.title/0",
    "value": {"value": "最後のタイトル", "language":"ja_JP"}
  }
]
```

Reply:
done